cms/urls.py

```diff
@@ -339,7 +339,7 @@
 
 # Content tagging
 urlpatterns += [
-    path('api/content_tagging/', include(('openedx.core.djangoapps.content_tagging.urls'))),
+    path('api/content_tagging/', include(('openedx.core.djangoapps.content_tagging.urls', 'oel_tagging'))),
```

Comment:
Ah, I see why we had to add the namespace manually.
I thought we had both the upstream API and the content tagging API registered separately, like this:

```python
path('api/content_tagging/', include(('openedx.core.djangoapps.content_tagging.urls'))),
path('api/tagging/', include(('openedx_tagging.core.tagging.rest_api.v1.urls'))),
```

But instead we are merging `oel_tagging` and `content_tagging` APIs into one endpoint. That should be fine for now, but in the future if we ever start tagging non-content things like users, we may need to better separate those APIs. Or I guess we could just add another "merged" endpoint like `/api/user_tagging` which includes `oel_tagging` and `user_tagging`. So yeah, I think this makes sense.

Reply:
Actually @yusuf-musleh what do you think about making this change to openedx-learning so that its views can be used with any namespace? https://github.com/open-craft/openedx-learning/commit/2d5fb28d9d57823da9f18c0b3c82ca1ef4fc4a0b

Reply:
Actually that diff can be improved even more using `url_namespace = request.resolver_match.namespace` rather than `url_namespace = request.resolver_match.view_name.split(":")[0]`

Reply:
@bradenmacdonald Oh thats pretty cool! Updated `openedx-learning` with your suggestion, including calling the namespace `content_tagging`, and it seems to work pretty well. 👍